.changeset/wet-fireants-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@whereby.com/browser-sdk": minor
```

Comment:
Could we change this to a patch release instead. This does not warrant a full minor release bump.

```suggestion
"@whereby.com/browser-sdk": patch
```